tests/integration_tests/test_wave_01_02_03.py

```diff
@@ -2,8 +2,8 @@
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
 
-@pytest.mark.skip
-@pytest.mark.integration_test
+# @pytest.mark.skip
+# @pytest.mark.integration_test
```

Comment:
It still passes in this case, but this second decorator ensures that the test gets run _after_ the unit tests, so you want to leave it as-is and don't comment it out.